packages/web/e2e/editCollection.test.ts

```diff
@@ -16,7 +16,7 @@ test('should persist collection edits', async ({ page }) => {
   await page.goto(permalink)
   await page
     .getByRole('link', { name: /edit album/i })
-    .click({ timeout: 20000 })
+    .click({ timeout: 20_000 })
```

Comment:
can you explain why this is needed

Reply:
it's not, just helped me visually confirm that it's 20s